eng/packaging.targets

```diff
@@ -84,8 +89,8 @@
   </ItemGroup>
 
   <!-- Add a package README file from. -->
-  <ItemGroup Condition="'$(PackageReadmeFile)' != ''">
-    <None Include="$(PackageReadmeFile)" Pack="true" PackagePath="\" />
+  <ItemGroup Condition="'$(PackageReadmeFilePath)' != ''">
+    <None Include="$(PackageReadmeFilePath)" Pack="true" PackagePath="\" />
```

Comment:
Is this where the path is prefixed? Maybe PackageReadmeFilePath should be named PackageReadmeFileName?

Reply:
Nope. This is just where the package readme file gets added as a package file so that it gets copied. The `PackageReadmeFile` property determines the path of the file in the package. This is a NuGet extension point, which is weird. This could have been implemented in a better way.